pkg/controller/backup_controller.go

```diff
@@ -58,6 +58,7 @@ type backupController struct {
 
 	backupper                pkgbackup.Backupper
 	lister                   listers.BackupLister
+	listerPodVolumes         listers.PodVolumeBackupLister
```

Comment:
let's call this `podVolumeBackupLister` or `pvbLister`, and update `lister` to be `backupLister`